builder/dockerfile/dispatchers.go

```diff
@@ -497,8 +497,11 @@ func healthcheck(b *Builder, args []string, attributes map[string]bool, original
 			}
 
 			healthcheck.Test = strslice.StrSlice(append([]string{typ}, cmdSlice...))
+		case "HTTP":
+			httpSlice := handleJSONArgs(args, attributes)
+			healthcheck.Test = strslice.StrSlice(append([]string{typ}, httpSlice...))
```

Comment:
It looks like the probe only uses the first argument. Perhaps there should be some kind of error reported here if the user passes multiple arguments. Also, do we need JSON support here? I think that's only useful for commands.